plonetheme/barceloneta/theme/less/contents.plone.less

```diff
@@ -37,9 +44,4 @@ body#visual-portal-wrapper.pat-plone #edit-zone {
 	.contenttype-collection:after {	content: '\e808';}
 	.contenttype-event:after {		content: '\e809';}
 	.contenttype-news-item:after {	content: '\e80f';}
-}
-
-//*// thumbnails for images, leadimages  in listings and portlets
-    .image-icon, .image-tile {
```

Comment:
Why are we removing this as well? Does this rule need a more specific selector maybe?